resources/migrations/000_migrations.yaml

```diff
@@ -3670,3 +3670,189 @@ databaseChangeLog:
                   name: result_metadata
                   type: text
                   remarks: 'Serialized JSON containing metadata about the result columns from running the query.'
+  - changeSet:
+      id: 58
+      author: arthur
+      changes:
+        - createTable:
+            tableName: field_fingerprint
+            remarks: 'tables are sampled periodically to capture a sample of the values in them. These snapshots are then used to make these statistics which will later be used to guess properties of that table'
```

Comment:
another bad idea:

copying and pasting docstrings from other pieces of code and not updating them, which leaves you with documentation that doesn't make sense